src/components/ui/checkbox.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
```

Comment:
for these ui tsx files.. follow the same naming convention na.. capitalize the file name? Card.tsx, Checkbox.tsx etc..